sdk/servicebus/Azure.Messaging.ServiceBus/tests/Processor/ProcessorTests.cs

```diff
@@ -362,6 +365,65 @@ public async Task CanRaiseEventsOnMockProcessor()
             Assert.IsTrue(processMessageCalled);
             Assert.IsTrue(processErrorCalled);
         }
+
+        [Test]
+        public async Task CanotStartProcessorWhenProcessorIsDisposed()
+        {
+            var mockConnection = GetMockedReceiverConnection();
+
+            var processor = new ServiceBusProcessor(
+                mockConnection,
+                "entityPath",
+                false,
+                new ServiceBusPlugin[] { },
+                new ServiceBusProcessorOptions());
+
+            processor.ProcessMessageAsync += _ => Task.CompletedTask;
+            processor.ProcessErrorAsync += _ => Task.CompletedTask;
+
+            await processor.DisposeAsync();
+
+            Assert.That(async () => await processor.StartProcessingAsync(),
+                Throws.InstanceOf<ObjectDisposedException>().And.Property(nameof(ObjectDisposedException.ObjectName)).EqualTo(nameof(ServiceBusProcessor)));
+        }
+
+        [Test]
+        public async Task CanotStartProcessorWhenConnectionIsClosed()
```

Comment:
```suggestion
        public async Task CannotStartProcessorWhenConnectionIsClosed()
```

Reply:
If there weren't typos, nobody would believe that I wrote it...